lib/matplotlib/figure.py

```diff
@@ -519,8 +519,8 @@ def add_axes(self, *args, **kwargs):
 
         Parameters
         ----------
-        rect : sequence of float
-            The dimensions [left, bottom, width, height] of the new Axes. All
+        rect : tuple (left, bottom, right, top)
+            The dimensions (left, bottom, right, top) of the new Axes. All
```

Comment:
Pretty sure this change is incorrect.

Reply:
What would be the correct way to put `[x_min, y_min, dx, dy]`?